### PR TITLE
Created a Team page. 

### PR DESCRIPTION
**Pull request by tusharrao198**
Issue #159 
1. Added a database model for team members which can be used to edit, add or update the information. 
2. Also added some media roots form images in settings.py file . 
3. Registered Models in order to access them from django-admin.
4. Added  Images shown on team page can now be shown ordered batchwise. 
5. Information about the links can be added later.

### CODE REVIEW

**Comment by LalitNM** [comment]
@tusharrao198 you need to rebase your branch.

**Comment by gajrajgchouhan** [comment]
You should use different branches for different PRs.

**Comment by tusharrao198** [comment]
> You should use different branches for different PRs, and do `git pull` before you start commiting to make sure your `clone` is up-to-date.

Yeah i did that but the previous PR was not merged until then. That why the rebase issue.

**Comment by harnaman-hk** [comment]
@tusharrao198  Can you attach a screenshot of how it looks on the frontend?

**Comment by tusharrao198** [comment]
Ok

**Comment by tusharrao198** [comment]
vlcsnap-2020-12-08-17h33m48s522

**Comment by tusharrao198** [comment]
1. Readme added for steps to add user through admin page. 
2. Directly add a member and his/her photo.
3. Batch column created in order to show images of the core team members  on the basis of Batch ( Badelog on the top) 
@harnaman-hk  , @gajrajgchouhan

File: src/web_main/settings.py [hunk 1]
```diff
@@ -124,3 +124,8 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+MEDIA_ROOT = os.path.join(BASE_DIR , 'stac_iitmandi/static/stac_iitmandi/images/')  
+MEDIA_URL  = '/'     
+
+#stac_iitmandi\static\stac_iitmandi\images
```
Comment (gajrajgchouhan) [inline src/web_main/settings.py:131]: This comment can be removed.

**Review by gajrajgchouhan — CHANGES_REQUESTED** [review]

File: src/web_main/settings.py (hunk 1 above)
Comment (tusharrao198) [reply src/web_main/settings.py:131]: ok

**Comment by tusharrao198** [comment]
Issue #162 resolved in last commit.

**Comment by tusharrao198** [comment]
image
image

**Comment by tusharrao198** [comment]
image

image

**Comment by gajrajgchouhan** [comment]
Thank you for the PR 🚀 !!

**Review by gajrajgchouhan — APPROVED** [review]

**Comment by LalitNM** [comment]
I strongly recommend that you put readme file in a directory with name `README.md` (in a directory where the contributor needs to set the database). Rest everything looks good :clap:

**Review by LalitNM — APPROVED** [review]

**Pull request merged**